src/index.html

```diff
@@ -9,6 +9,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a class="logo" href="#home">
+        <img class="logo__img"
+             src="images/logo.png"
+             alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__links is-active"
+               href="#apple">Apple</a>
```

Comment:
[Check yourself](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) and fix all cases 
![image](https://github.com/mate-academy/layout_moyo-header/assets/70009205/b9d576d9-83b6-4091-b263-bda714a33a60)

```suggestion
            <a 
              class="nav__links is-active"
              href="#apple"
            >
              Apple
            </a>
```